relay-dynamic-config/src/defaults.rs

```diff
@@ -260,6 +264,10 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
                 Tag::with_key("resource.render_blocking_status")
                     .from_field("span.sentry_tags.resource.render_blocking_status")
                     .when(is_resource.clone()),
+                // HTTP module:
+                Tag::with_key("span.status_code")
+                    .from_field("span.sentry_tags.status_code")
```

Comment:
Do we need the tag in `d:spans/exclusive_time_light@millisecond` too?

Reply:
Good point.

Reply:
Unsure what you mean—from what I can see I added the tag to that metric, did I do something wrong?

Reply:
The tag is added to two metrics: `d:spans/exclusive_time@millisecond` and `d:spans/exclusive_time_light@millisecond`. Do we need the tag in both metrics, or just one of the two metrics is good enough?

Reply:
Oh I see! I need it for both, since the `light` metric is only used on the landing page, and the regular one is used on the domain landing pages, where we have to break down by transaction

Reply:
Sounds good! :shipit: